src/css/style.css

```diff
@@ -0,0 +1,196 @@
+body {
+  font-family: "Lato", sans-serif;
+}
+
+.container {
+  width: 94%;
```

Comment:
##### **[OPTIONAL]**
Set container width to 100%; As it stands now this how it looks on my PC
![Screenshot from 2019-09-26 19-23-41](https://user-images.githubusercontent.com/29695661/65731338-442b1800-e093-11e9-850a-102eb6463bc0.png)